modules/databend/src/test/java/org/testcontainers/databend/DatabendContainerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.testcontainers.databend;
+
+import org.junit.Test;
+import org.testcontainers.db.AbstractContainerDatabaseTest;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DatabendContainerTest extends AbstractContainerDatabaseTest {
+
+    @Test
+    public void testSimple() throws SQLException {
+        try (DatabendContainer databend = new DatabendContainer("datafuselabs/databend:v1.2.623-nightly")) {
+            databend.start();
+
+            ResultSet resultSet = performQuery(databend, "SELECT 1");
+
+            int resultSetInt = resultSet.getInt(1);
+            assertThat(resultSetInt).isEqualTo(1);
+        }
+    }
+
+    @Test
+    public void customCredentialsWithUrlParams() throws SQLException {
+        try (
+            DatabendContainer databend = new DatabendContainer("datafuselabs/databend:v1.2.623-nightly")
+                .withUsername("databend")
+                .withPassword("databend")
+                .withDatabaseName("default")
+                .withUrlParam("ssl", false)
+        ) {
+            databend.start();
+
+            ResultSet resultSet = performQuery(
+                databend,
+                "SELECT 1'"
+            );
+
+            int resultSetInt = resultSet.getInt(1);
+            assertThat(resultSetInt).isEqualTo(1);
```

Comment:
add asserting to test url param is included in jdbc url

```java
assertThat(databend.getJdbcUrl).contains("?ssl=false");
```

Reply:
@hantmac I don't see this change

Reply:
Fixed. Thanks!